src/analysis/souffle/taint.dl

```diff
@@ -20,11 +20,19 @@
 #include "authorization_logic.dl"
 #include "dataflow_graph.dl"
 #include "tags.dl"
+#include "operations.dl"
 
 //-----------------------------------------------------------------------------
 // Relation Declarations
 //-----------------------------------------------------------------------------
 
+// This fact indicates that, in some unspecified way, accessPath is influenced
+// by influencer. Unlike `resolvedEdge`, which specifically represents an exact
+// assignment, this can represent any dataflow from any operation. Should not
+// be populated directly, is populated indirectly by edges and other
+// operations.
+.decl influencedBy(owner: Principal, accessPath: AccessPath, influencer: AccessPath)
```

Comment:
`InfluencedBy` is not a standard term and this is more like controlDependence. Can we say `controlDependentOn`, `controlDependenceEdge`,  or something like that?

Actually based on the rule below, it looks like this more like dataflow or direct data dependence. If so, why can't we use `edge` itself?

Reply:
We were discussing changing the meaning of `edge` to mean "assignment", or an exact copy of one value from one location into another. This is attempting to capture the fact that taint may flow from the `influencer` to the target `accessPath`, and is a specific concept for confidentiality tag flow.

Reply:
If there is no special semantics for an operator, we should just generate `edge` relations. 

Didn't we think of an edge from a special access path to capture the fact that an output value may be created without depending on any input values?

Reply:
Yes, that does make sense. I can refactor this PR to remove `influencedBy` and add a special `NoIntegrity` `AccessPath` to block integrity propagation.

Reply:
Done.